Makefile

```diff
@@ -30,3 +30,13 @@ test-carthage:
 # Tests if current branch ships a valid Cocoapods project.
 test-cocoapods:
 		@cd dependency-manager-tests/cocoapods && $(MAKE)
+
+newversion:
+		@read -p "Enter version number: " version;  \
+		echo "// GENERATED FILE: Do not edit directly\n\ninternal let sdkVersion = \"$$version\"" > Sources/Datadog/Versioning.swift; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDK.podspec.meta > DatadogSDK.podspec; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDKObjc.podspec.meta > DatadogSDKObjc.podspec; \
+		git add . ; \
+		git commit -m "Bumped version to $$version"; \
+		git push -u origin master; \
```

Comment:
i'm not sure if that'll work, `master` is protected
i think we should do the push explicitly/manually

Reply:
maybe we can split bumping and shipping into 2 tasks
the code above can be `bump`
and we can push to `pod trunk` and Github with `ship`

what do you think?

Reply:
it sounds good, I will work on it